examples/simpleInvoke/main.go

```diff
@@ -3,65 +3,63 @@ package main
-	privateKey            string = "0x0687bf84896ee63f52d69e6de1b41492abeadc0dc3cb7bd351d0a52116915937" //Replace it with your account private key
-	public_key            string = "0x58b0824ee8480133cad03533c8930eda6888b3c5170db2f6e4f51b519141963"  //Replace it with your account public key
-	someContract          string = "0x4c1337d55351eac9a0b74f3b8f0d3928e2bb781e5084686a892e66d49d510d"   //Replace it with the contract that you want to invoke
-	contractMethod        string = "increase_value"                                                     //Replace it with the function name that you want to invoke
+	someContract   string = "0x0669e24364ce0ae7ec2864fb03eedbe60cfbc9d1c74438d10fa4b86552907d54" //Replace it with the contract that you want to invoke. In this case, an ERC20
+	contractMethod string = "mint"                                                               //Replace it with the function name that you want to invoke
 )
 
 func main() {
-	// Loading the env
-	godotenv.Load(fmt.Sprintf(".env.%s", name))
-	url := os.Getenv("INTEGRATION_BASE") //please modify the .env.testnet and replace the INTEGRATION_BASE with a starknet goerli RPC.
-	fmt.Println("Starting simpleInvoke example")
-
-	// Initialising the connection
-	clientv02, err := rpc.NewProvider(url)
+	// Load variables from '.env' file
+	rpcProviderUrl := setup.GetRpcProviderUrl()
+	accountAddress := setup.GetAccountAddress()
+	accountCairoVersion := setup.GetAccountCairoVersion()
+	privateKey := setup.GetPrivateKey()
+	publicKey := setup.GetPublicKey()
+
+	// Initialize connection to RPC provider
+	client, err := rpc.NewProvider(rpcProviderUrl)
 	if err != nil {
-		log.Fatal(fmt.Sprintf("Error dialing the RPC provider: %s", err))
+		panic(fmt.Sprintf("Error dialing the RPC provider: %s", err))
 	}
 
+	// Initialize the account memkeyStore (set public and private keys)
+	ks := account.NewMemKeystore()
+	privKeyBI, ok := new(big.Int).SetString(privateKey, 0)
+	if !ok {
+		panic("Fail to convert privKey to bitInt")
+	}
+	ks.Put(publicKey, privKeyBI)
+
 	// Here we are converting the account address to felt
-	account_address, err := utils.HexToFelt(account_addr)
+	accountAddressInFelt, err := utils.HexToFelt(accountAddress)
 	if err != nil {
-		panic(err.Error())
+		fmt.Println("Failed to transform the account address, did you give the hex address?")
+		panic(err)
 	}
-	// Initializing the account memkeyStore
-	ks := account.NewMemKeystore()
-	fakePrivKeyBI, ok := new(big.Int).SetString(privateKey, 0)
-	if !ok {
-		panic(err.Error())
+	// Initialize the account
+	accnt, err := account.NewAccount(client, accountAddressInFelt, publicKey, ks, accountCairoVersion)
+	if err != nil {
+		panic(err)
 	}
-	ks.Put(public_key, fakePrivKeyBI)
 
 	fmt.Println("Established connection with the client")
 
 	// Here we are setting the maxFee
-
 	maxfee, err := utils.HexToFelt("0x9184e72a000")
```

Comment:
nit: I had to raise the maxFee to 0x9184e72a00000 for this to work

Reply:
This was due to high gas price volatility. I fixed this in the new commit by updating the transaction's max fee field with the value returned by EstimateFee + 20%